_posts/2014-02-20-first-new-robot-2014-hollie.markdown

```diff
@@ -17,4 +17,4 @@ MoveIt! is excited to welcome our first new robot in 2014! HoLLiE the Assistance
 ![](/assets/images/robots/hollie.png)
 
 
-We would like to thank Andreas Hermann from Interaktive Diagnose- und Servicesysteme (IDS) at FZI for sending us information about HoLLiE!
+We would like to thank Andreas Hermann from Interactively Diagnose- und Servicesysteme (IDS) at FZI for sending us information about HoLLiE!
```

Comment:
This german term shouldn't be translated.

Reply:
I updated this pr to ignore translating those words